src/server/pkg/pachhash/hash.go

```diff
@@ -0,0 +1,34 @@
+package pachhash
+
+import (
+	"encoding/hex"
+	"hash"
+
+	"golang.org/x/crypto/blake2b"
+)
+
+// OutputSize is the size of an Output in bytes
+const OutputSize = 32
+
+// Output is the output of the hash function.
+// Sum returns an Output
+type Output = [OutputSize]byte
+
+// New creates a new hasher.
+func New() hash.Hash {
+	h, err := blake2b.New256(nil)
+	if err != nil {
+		panic(err)
```

Comment:
This is getting out of hand lol. Maybe we should just change the signature to return an error?

Reply:
An error from this constructor is a configuration error. We want to know as soon as possible, and there is no point in checking for it everywhere because there is no way to handle it, other than by using the constructor correctly.

Reply:
https://pkg.go.dev/golang.org/x/crypto/blake2b#New256
If you give a key that is too large it errors.